hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/master/MasterStatusTmpl.jamon

```diff
@@ -604,7 +604,6 @@ AssignmentManager assignmentManager = master.getAssignmentManager();
         <th>CLOSED</th>
         <th>CLOSING</th>
         <th>OFFLINE</th>
-        <th>FAILED</th>
```

Comment:
Why drop failed reporting?

Reply:
There is no single 'FAILED' status; there is FAILED_CLOSE and FAILED_OPEN. I do not seem to be able to pass two STATEs to filter on  so in the meantime, just lump these in w/ the 'Other' states for now.